packages/ocr_extraction/src/server.py

```diff
@@ -62,7 +62,11 @@ async def process_and_predict(file: UploadFile = File(...), llm_model: str = For
     logging.info("In predict")
     file_bytes = await file.read()
     logging.info(f"File length = {len(file_bytes)}")
-    docstream = DocumentStream(name="test", stream=BytesIO(file_bytes))
+
+    filename = "document.txt"
+    if file.filename is not None:
+        filename = file.filename
```

Comment:
Why we need to hardcode the value here?
Shouldn't the taskrunner provide a document already?